docs/build.md

```diff
@@ -0,0 +1,61 @@
+# Installation and Building
+
+## Prerequisites
+
+In order to build Wallet from the source, first make sure to install the following:
+
+- [Go 1.17](https://go.dev/doc/install)
+- [Npm 8](https://docs.npmjs.com/cli/v8/configuring-npm/install)
+- [Node 17](https://nodejs.org/)
```

Comment:
I think its better to point to LTS versions - current LTS is v16 https://nodejs.org/en/about/releases/